packages/shared/src/utils/validation/regex-validation.ts

```diff
@@ -1,8 +1,8 @@
 export const regex_checks = {
     address_details: {
         address_city: /^\p{L}[\p{L}\s'.-]{0,99}$/u,
-        address_line_1: /^([a-zA-Z0-9'.,:;()@#/-]+\s)*[a-zA-Z0-9'.,:;()@#/-]{1,70}$/,
-        address_line_2: /^([a-zA-Z0-9'.,:;()@#/-]+\s)*[a-zA-Z0-9'.,:;()@#/-]{0,70}$/,
+        address_line_1: /^([a-zA-Z0-9’'.,:;()@#/-]+\s)*[a-zA-Z0-9’'.,:;()@#/-]{1,70}$/,
```

Comment:
I still can see this error in the test link
![image](https://user-images.githubusercontent.com/82078941/219062170-a2e1e04c-5a5d-4149-99ae-cdc4ff71d2a6.png)
 you also need to add this rule in the` /account/personal-details`

Reply:
I think the card wants this specific symbol `’` not the `

<img width="1728" alt="Screenshot 2023-02-16 at 7 59 01 PM" src="https://user-images.githubusercontent.com/103104395/219359202-95e91e32-443b-4983-9301-75e529f72bab.png">

**The Card Description**
While creating deriv real account, on `Complete your address details` part, `First line of address` and  `Second line of address` answer fields are not allowing user to fill in addresses with `’` apostrophe

**Expected behavior from the card:**
`First line of address` and  `Second line of address` answer fields should allow user to enter address details including `’` apostrophe